workers/shared/patterns/notification/helper.py

```
@@ -18,6 +19,102 @@
+                "additional_data": payload.additional_data or {},
+            },
+            timeout=10,
+        )
+        logger.info(
+            "Enqueued BATCHED notification %s for pipeline %s execution %s",
+            notification["id"],
+            payload.pipeline_id,
+            payload.execution_id,
+        )
+        return True
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)